project-base/storefront/components/Layout/CommonLayout.tsx

```diff
@@ -46,6 +51,8 @@ export const CommonLayout: FC<CommonLayoutProps> = ({
                 defaultDescription={description}
                 defaultHreflangLinks={hreflangLinks}
                 defaultTitle={title}
+                ogImageUrlDefault={ogImageUrlDefault ?? undefined}
```

Comment:
Why do you have `?? undefined` here? I thought the only option is to have a string or undefined 🤔 that means you could just pass `ogImageUrlDefault`, or not?

Reply:
Correct.